### ISSUE TITLE
Allow add --path to pull from http server

### ISSUE THREAD

**Issue by iwoloschin** [sdispater/poetry#1200]
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [documentation](https://poetry.eustace.io/docs/) and believe that my question is not covered.

## Feature Request
I have a private dependency for my project that ships as a tarball.  Currently it is installed using `pip`'s native ability to install direct from an http path:
```
pip install http://private-server.localdomain/path/to/tarball.tar.gz
```

If I downloaded the tarball I think poetry could use it:
```
wget http://private-server.localdomain/path/to/tarball.tar.gz
poetry add --path ./tarball.tar.gz package_name
```
Would it be possible to add support to allow poetry to pull the file direct from the server, something like this:
```
poetry add --path http://private-server.localdomain/path/to/tarball.tar.gz package_name
```

**Comment by sdispater** [comment]
This has been added in #1260. It will be available in the `1.0.0` release.

**Comment by marctorrellas** [comment]
> This has been added in #1260. It will be available in the `1.0.0` release.

when is the release expected?
